### ISSUE TITLE
Build for web browsers?

### ISSUE THREAD

**Issue by loredanacirstea** [cabal-club/cabal-client#75]
I know cabal is based on Dat and Dat has an sdk that is supposed to work for browsers.
Would cabal-client work for browsers too? Any plans to provide a build?

**Comment by cblgh** [comment]
@loredanacirstea yes! that's actually one of my next pet projects within cabal, now that [channel archiving](https://github.com/cabal-club/cabal-client/pull/74) has been merged!

you can see some early explorations of this in [caballo](https://github.com/cblgh/caballo) :)
